java/jsinterop/base/BUILD

```diff
@@ -36,7 +36,7 @@ genrule(
     srcs = glob(["**/*.java"]),
     outs = ["base-j2cl.srcjar"],
     cmd = "\n".join([
-        "sed -i 's/\/\/J2CL_ONLY\ //g' $(SRCS)",
+        "sed -i '' 's/\/\/J2CL_ONLY\ //g' $(SRCS)",
```

Comment:
This does not work on Linux. At least with sed version 4.4.

We should probably avoid -i and just use > and cp.

Reply:
you need to use `sed -i.bak 's/\/\/J2CL_ONLY\ //g' $(SRCS)` for being compatible both in linux and Mac OsX

It's similar what I did in https://github.com/google/jsinterop-generator/commit/bbbd78cc0b682951d4fcd284ba2871d27aa7eec7

Reply:
Odd, I'm running sed 4.5, and I know I've used this trick in years past...

I'll try Julien's suggestion on both platforms.